functorch/_src/make_functional.py

```diff
@@ -44,7 +44,7 @@ def _get_nested_attr(obj: nn.Module, names: List[str]) -> None:
     if len(names) == 1:
         return getattr(obj, names[0])
     else:
-        _get_nested_attr(getattr(obj, names[0]), names[1:])
+        return _get_nested_attr(getattr(obj, names[0]), names[1:])
```

Comment:
Add a test? (our tests for make_functional are in test/functorch/test_eager_transforms

Reply:
Perfect, thanks!

Reply:
done in last commit